logspec-worker/logspec_api.py

```diff
@@ -125,13 +127,25 @@ def create_special_boot_error(summary):
     return errors_list, new_status
 
 
-def new_issue(logspec_error, test_type, origin):
+def new_issue(cursor, logspec_error, test_type, origin):
```

Comment:
I suggest we rename this method to something more fitting with the new behavior.
In django we have "get_or_create" to achieve something similar.